### PR TITLE
[Live] Migrate to charts-build-scripts v0.2.1

### PR DESCRIPTION
**Pull request by aiyengar2**
Switches scripts from old Bash scripts to new Go scripts based on https://github.com/rancher/charts-build-scripts

Related Issue: https://github.com/rancher/rancher/issues/31500

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -809,9 +809,9 @@ entries:
       catalog.cattle.io/release-name: portshift-operator
     apiVersion: v1
     appVersion: v0.1.3
-    created: "2020-10-19T19:35:34.247083742Z"
-    description: 'Portshift cloud-native security platform is an agentless security
-      solution for containerized applications '
+    created: "2021-06-23T17:44:55.488041-07:00"
+    description: |
+      Portshift cloud-native security platform is an agentless security solution for containerized applications
```
Comment (aiyengar2) [inline index.yaml:814]: minor change that generates the description in a new line. The other changes are just timestamp changes.

File: assets/index.yaml [hunk 2]
```diff
@@ -1,917 +0,0 @@
-apiVersion: v1
```
Comment (aiyengar2) [inline assets/index.yaml:1]: This file was never necessary and has therefore been removed.

**Comment by aiyengar2** [comment]
@PennyScissors this PR is ready for review.

**Review by PennyScissors — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Pull request merged**